src/asm2wasm.h

```diff
@@ -879,6 +878,76 @@ class Asm2WasmBuilder {
     return ret;
   }
 
+  Expression* makeTrappingFloatToInt64(bool signed_, Expression* value) {
```

Comment:
It feels like we should be able to reuse a lot of the logic from `makeTrappingFloatToInt32`, they're very structurally similar.